### DISCUSSION TITLE
Cydia Store Support

### DISCUSSION THREAD

**Discussion by xxHoBoZxx** [wstyres/Zebra#693] [Ideas]
**Describe the bug**
When signing into Cydia account using Google login, it prompts to "Open link in Zebra" instead of "Open in Cydia".

**To Reproduce**
Have Cydia and Zebra installed, from Cydia try to login to account with Google, it prompts to "Open in Zebra".

**Expected behavior**
It should prompt to "Open in Cydia".

**Screenshots**
![IMG_0016](https://user-images.githubusercontent.com/32913594/70855513-86ddde80-1e91-11ea-9350-b10e7b79fe23.png)

**Device Information (please complete the following information):**
 - iOS Version 13.2.3: 
 - Zebra Version 1.0~beta33: 
 - Device Type iPhone 8: 
 - Jailbreak Type checkra1n: 

**Crash Log or System Log**
Please provide a Crash Log to help us solve the issue.

**Additional context**
Only way to correct is to uninstall Zebra and login in Cydia and then re-install Zebra. Not that I can use or buy anything in Cydia. LOL

**Comment by wstyres** [comment]
Unfortunately this is a bit of an issue that is hard to deal with. We can't really pick and choose which URLs we open from cydia:// so it is all or none of them. An alternative I might suggest is maybe somehow we can implement the Cydia Store login into Zebra.

**Comment by xxHoBoZxx** [comment]
> 
> 
> Unfortunately this is a bit of an issue that is hard to deal with. We can't really pick and choose which URLs we open from cydia:// so it is all or none of them. An alternative I might suggest is maybe somehow we can implement the Cydia Store login into Zebra.

I understand. I only tried to login to Cydia just to see if it would work again. Just wanted to let you know about it.

**Comment by wstyres** [comment]
I’ll just consider this as a feature request for downloading from the Cydia Store then
